src/sync.py

```diff
@@ -14,27 +15,25 @@ def sync_board(source: 'Board', sink: 'Board'):
 
         if source.__class__.__name__ == 'ZenHubRepo' and sink.__class__.__name__ == 'JiraRepo':
             for key, issue in source.issues.items():
-                print(f'syncing {sink.issues[issue.jira_key]} from {key}')
+                logging.info(f'syncing {sink.issues[issue.jira_key]} from {key}')
```

Comment:
`sink.issues[issue.jira_key]` seems to be an object and doesn't return the key as a string.

Reply:
Fixed